guides/v2.13.0/controllers/index.md

```diff
@@ -1,6 +1,6 @@
 ## Controllers
 
-Controllers behave like a specialized type of Component that is rendered by
+Controllers behave like a specialised type of Component that is rendered by
```

Comment:
This one, let's keep it spelled with a "z". I think the guides follow the American spelling in the case of s/z variation.

```suggestion
Controllers behave like a specialized type of Component that is rendered by
```